frontend/packages/dev-console/src/components/cluster-tasks/ClusterTaskDetailsPage.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+// FIXME upgrading redux types is causing many errors at this time
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { useSelector } from 'react-redux';
+import { DetailsPageProps, DetailsPage } from '@console/internal/components/factory';
+import { getActiveNamespace } from '@console/internal/reducers/ui';
+import { RootState } from '@console/internal/redux';
+import { navFactory, Kebab } from '@console/internal/components/utils';
+import { DetailsForKind } from '@console/internal/components/default-resource';
+import { usePipelinesBreadcrumbsFor } from '../pipelines/hooks';
+
+const ClusterTaskDetailsPage: React.FC<DetailsPageProps> = (props) => {
+  const { kindObj, match, kind } = props;
+  const activeNamespace = useSelector((state: RootState) => getActiveNamespace(state));
```

Comment:
I think you can avoid doing this out here if you just bake this functionality into the hook.

This is a bit of annoying usage due to the 3 imports + ts-ignore override.